src/components/Pressable/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@further-ventures/fv-react-ui-components.button",
```

Comment:
pressable instead of button ?

Reply:
But also do we really need package.json for this one ?

Reply:
Nope, we don't. Good point